### ISSUE TITLE
运用miniPaint搭建了一套中文版在线ps网页，为什么有的单词翻译的不准确，比如guide

### ISSUE THREAD

**Issue by geeeeeeeek** [viliusle/miniPaint#388]
在线ps网页中文版。https://zaixianps.net

**Comment by phpony** [comment]
Well I don't see the translation for "Guides" [here in zh lang file](https://github.com/viliusle/miniPaint/blob/d36fbadd86e49fbc62152e6f5c718b917f6769fe/src/js/languages/zh.json#L1).

You probably should add some missing strings into this file and build your app again. Also check [this pull request](https://github.com/viliusle/miniPaint/pull/386) - without this patch some parts of the interface can't be translated at all.

**Comment by phpony** [comment]
Yep. There is a bug. Fixed in https://github.com/viliusle/miniPaint/pull/386 but yet not merged.

**Comment by viliusle** [comment]
Already merged (on master branch).

**Comment by geeeeeeeek** [comment]
you can use it: https://freeps.org/

**Comment by viliusle** [comment]
> > > 在线ps网页中文版。https://zaixianps.net
> > 
> > 
> > 我把语言设置成中文但是需要点击菜单才会变成中文，不点击就还是英文。这地方是不是有bug
> 
> 我也是一样，在miniPaint/src/js/config.js中，设置config.LANG = 'zh';并不生效。只能在菜单tools-languages-简体中文，这样子设置。无法设置默认语言

You can start minipaint in `zh` language by default: https://viliusle.github.io/miniPaint/?lang=zh

**Comment by iy3k** [comment]
到底是为什么？
